web/src/main/java/org/mskcc/cbio/portal/web/api/ApiController.java

```diff
@@ -233,7 +234,7 @@
             notes = "")
     @Transactional
     @RequestMapping(value = "/geneticprofiledata", method = {RequestMethod.GET, RequestMethod.POST})
-    public @ResponseBody List<DBProfileData> getGeneticProfileData(
+    public @ResponseBody List<Serializable> getGeneticProfileData(
```

Comment:
They are already applied, I think you are confused because I didn't use @View annotation, it’s just to create a different view of models for different services, which I didn’t need for that service, because we don’t need to hide anything for that web_api service specifically or have a different property name just for that web_api service.